src/main/scala/temple/builder/DatabaseBuilder.scala

```diff
@@ -24,10 +24,15 @@ object DatabaseBuilder {
   private def toColDef(name: String, attribute: Attribute): ColumnDef = {
     val nonNullConstraint = when(!attribute.valueAnnotations.contains(Nullable)) { ColumnConstraint.NonNull }
 
+    val primaryKeyConstraint = attribute match {
+      case IDAttribute => Some(ColumnConstraint.PrimaryKey)
+      case _           => None
+    }
```

Comment:
inclined to suggest `when(attribute == IDAttribute) { ColumnConstraint.PrimaryKey }`, but up to you